### DISCUSSION TITLE
不是，这个怎么解啊（

### DISCUSSION THREAD

**Discussion by HDGOLJ** [share121/inter-knot#4734] [崩坏：星穹铁道]
我玩了好久都没解开，过主线时被硬控了<img src="https://img-static.mihoyo.com/communityweb/upload/59fe234835e808c41b0208c50be44914.png" alt="偶像姬-大声bb" width="75" >
![image](https://github.com/user-attachments/assets/3b19372b-50cc-464e-b169-c3cb63250d13)
![image](https://github.com/user-attachments/assets/8a143a88-84f0-4383-afc7-00556cf44f50)

**Comment by wodedunpaifeichangying** [comment]
从最上面突出一格那里开始搭两格，三个格子竖着连的最下面一个格子一格
